### ISSUE TITLE
Auth /me call blocks rendering of content

### ISSUE THREAD

**Issue by simplenotezy** [nuxt-community/auth-module#635]
### Version

[v4.8.5](https://github.com/auth-module/releases/tag/v4.8.5)

### Reproduction link

[https://v2.thejewelleryroom.com/product/rebecca-ring](https://v2.thejewelleryroom.com/product/rebecca-ring)

### Steps to reproduce

I have noticed that the /me call produced by "fetchUser". This line:

	  async fetchUser (endpoint) {
	    // Token is required but not available
	    if (this.options.tokenRequired && !this.$auth.getToken(this.name)) {
	      return
	    }

	    // User endpoint is disabled.
	    if (!this.options.endpoints.user) {
	      this.$auth.setUser({})
	      return
	    }

	    // Try to fetch user and then set
	    const user = await this.$auth.requestWith(
	      this.name,
	      endpoint,
	      this.options.endpoints.user
	    )
	    this.$auth.setUser(user)
	  }

Have a look at this gif, that illustrates that rest of the content is first loaded when /me call has completed. The content-block being rendered has nothing to do with the auth module, so it is pretty weird.

![ezgif-3-db52a2bfd40a.gif](https://cmty.s3.eu-west-1.amazonaws.com/102371998/8ed7b2e0-83c8-11ea-828f-5fb75bca9643.gif)

### What is expected ?

API Call to /me (fetchUser) should not block any content form being rendered

### What is actually happening?

The API call blocks rendering of the page. The page will first render once the call has completed.

<div align="right"><sub><em>This bug report is available on <a href="https://cmty.app/nuxt">Nuxt</a> community (<a href="https://cmty.app/nuxt/auth-module/issues/c555">#c555</a>)</em></sub></div>

**Comment by bramburn** [comment]
maybe an async is not setup somewhere that's generating this error. I had something similar with something else last night.

**Comment by MrJmpl3** [comment]
Please, update the module and check if this error persists.

**Comment by simplenotezy** [comment]
I have just updated, unfortunately it still happens :/

Check here: https://v2.thejewelleryroom.com/product/offspring-ring-2

**Comment by MrJmpl3** [comment]
Interesting , The console stuck in "Making API get request to  /wp/v2/users/me  params" , @simplenotezy don't have a editable example? Like codepen to isolate the problem?

**Comment by simplenotezy** [comment]
@MrJmpl3 yeah you should see that console message if you are logged in (e.g. created a test account). If you are not logged in though, it shouldn't appear. It's just a simple onRequest method for debugging purposes:

	$axios.onRequest((config) => {
		console.log('Making API ' + config.method + ' request to ', config.url, ' params ', config.params);
	});

I'm sorry, but I don't have a codepen. I could grant you access to our repo if you'd like.

**Comment by simplenotezy** [comment]
Hmm.. I noticed I had wrapped a lot of parts of the product page inside <client-only> which was causing the delay. Still weird that the /me would block the rendering of the <client-only> contents. Like the client-only content would first get loaded once /me call has completed.
